src/components/authpage/validation/auth-schemas.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from 'zod';
+
+// 공통 정규식
+const passwordRegex =
+  /^(?=.*[a-zA-Z])(?=.*\d)(?=.*[@$&*?!%])[A-Za-z\d!@$%&*?]{8,15}$/;
+
+// 로그인 스키마
+export const loginSchema = z.object({
+  email: z
+    .string()
+    .nonempty({ message: '이메일을 입력해주세요' })
```

Comment:
<img width="536" alt="image" src="https://github.com/user-attachments/assets/213e827b-7b4d-4935-b22a-3408c319b8ca">

- nonempty 더이상 사용하지 않는 문법인거같아요. 아래처럼 수정하는게 좋을것 같습니다!

```suggestion
    .min(1, { message: '이메일을 입력해주세요' })
```